### DISCUSSION TITLE
Certificate (report), log on the Ventoy Key

### DISCUSSION THREAD

**Discussion by CreaTech56** [PartialVolume/shredos.x86_64#226] [Q&A]
My idea of the day
I use Ventoy with different images, originals and mods (loadkeys and nwipe options)
For having certificate on the ventoy key, during the process of creating the key, i preserve some space at the end of the disk and i formated it in FAT32 ( Ventoy  : Option : Partition Configuration)
I tried it and it seems to work well

**Comment by PartialVolume** [comment]
That's one of the advantages of using Ventoy instead of dd, that the partition can fill the whole of the USB device giving you plenty of storage.

The next release of ShredOS will save/restore to exfat & fat16 formatted drives as well as FAT32, making it a little bit more flexible.
